<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExamResultManagement</class>
 <widget class="QDialog" name="ExamResultManagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Exam Result Management</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#ExamResultManagement {
       background-color:#1B0633;
border:5px solid #B89AD8;
        border-radius: 10%;
}</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>691</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
border-radius: 10%;
     font-size: 19px;
        color: #E8D4F4 ;
font-weight:700;}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="studentIdInput">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit#studentIdInput {
     color:#1B0633;
    border: 3px solid #8d4d95;
    border-radius: 10px;
        font-size: 17px;
font-weight: 570;
padding: 0px 8px 0px 8px;
background-color: #E8D4F4 ;
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSubject">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#labelSubject {
        font-size: 19px;
        color: #E8D4F4 ;
font-weight:700;
}</string>
    </property>
    <property name="text">
     <string>Subject:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="marksInput">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>40</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit#marksInput {
     color:#1B0633;
    border: 3px solid #8d4d95;
    border-radius: 10px;
        font-size: 17px;
font-weight: 570;
padding: 0px 8px 0px 8px;
background-color: #E8D4F4 ;
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="studentNameInput">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit#studentNameInput {
       color:#1B0633;
    border: 3px solid #8d4d95;
    border-radius: 10px;
        font-size: 17px;
font-weight: 570;
padding: 0px 8px 0px 8px;
background-color: #E8D4F4 ;
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelClass">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#labelClass {
         font-size: 19px;
        color: #E8D4F4 ;
font-weight:700;
}</string>
    </property>
    <property name="text">
     <string>Class:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelStudentName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#labelStudentName {
       font-size: 19px;
        color: #E8D4F4 ;
font-weight:700;
}</string>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="classInput">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit#classInput {
     color:#1B0633;
    border: 3px solid #8d4d95;
    border-radius: 10px;
        font-size: 17px;
font-weight: 570;
padding: 0px 8px 0px 8px;
background-color: #E8D4F4 ;
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelStudentId">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#labelStudentId {
        font-size: 19px;
        color: #E8D4F4 ;
font-weight:700;
}</string>
    </property>
    <property name="text">
     <string>ID:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="subjectInput">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit#subjectInput {
   color:#1B0633;
    border: 3px solid #8d4d95;
    border-radius: 10px;
        font-size: 17px;
font-weight: 570;
padding: 0px 8px 0px 8px;
background-color: #E8D4F4 ;
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="totalmarksInput">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>80</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit#totalmarksInput {
   color:#1B0633;
    border: 3px solid #8d4d95;
    border-radius: 10px;
        font-size: 17px;
font-weight: 570;
padding: 0px 8px 0px 8px;
background-color: #E8D4F4 ;
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMarks">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#labelMarks {
         font-size: 19px;
        color: #E8D4F4 ;
font-weight:700;
}</string>
    </property>
    <property name="text">
     <string>Marks:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTotalMarks">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>70</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#labelTotalMarks {
      font-size: 19px;
        color: #E8D4F4 ;
font-weight:700;
}</string>
    </property>
    <property name="text">
     <string>Total Marks:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>120</y>
      <width>171</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#clearButton {
     border: 1.5px solid #B89FD0;
    border-radius: 10px;
    padding: 6px;
    font-size: 16px;
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                            stop:0 rgb(95, 2, 95),
                            stop:1 rgb(216, 181, 221));
color:#e4c9f2;
font-weight:700;
}

/* Hover effect */
QPushButton#clearButton:hover {
 background-color: #69166f;
}

</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updateButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>120</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#updateButton {
  border: 1.5px solid #B89FD0;
    border-radius: 10px;
    padding: 6px;
    font-size: 16px;
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                            stop:0 rgb(95, 2, 95),
                            stop:1 rgb(216, 181, 221));
color:#e4c9f2;
font-weight:700;
}

/* Hover effect */
QPushButton#updateButton:hover {

  background-color: #69166f;
}
</string>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>171</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#saveButton {
     border: 1.5px solid #B89FD0;
    border-radius: 10px;
    padding: 6px;
    font-size: 16px;
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                            stop:0 rgb(95, 2, 95),
                            stop:1 rgb(216, 181, 221));
color:#e4c9f2;
font-weight:700;
}

/* Hover effect */
QPushButton#saveButton:hover {
   background-color: #69166f;
}

</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableView" name="tableViewResults">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>671</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableView#tableViewResults {
border: 1.5px solid #b0b0b0;  /* Softer grey border */
    border-radius: 10px;
    background-color: #d4d4d4;    /* Lighter table background */
    font-family: 'Segoe UI', sans-serif;
    font-size: 11pt;
    color: #1f1f1f;
    gridline-color: #bcbcbc;      /* Mid grey grid lines */
    alternate-background-color: #e2e2e2;  /* Subtle row alternation */
    selection-background-color: #b0b0b0;  /* Visible selection */
    selection-color: #000000;
    font-weight: 500;

}

QHeaderView::section {
    background-color: #4b4b4b;    /* Darker grey header */
    color: #ffffff;               /* Light font for contrast */
    font-weight: bold;
    font-size: 11pt;
    padding: 6px;
    border: none;
    font-family: 'Segoe UI', sans-serif;
}

QTableWidget QTableCornerButton::section {
   background-color: #8a8a8a;    /* Match header color */
    border: none;
}

QTableWidget::item {
      padding: 8px 10px;
    font-size: 10pt;
    color: #1f1f1f;
    background-color: transparent;
}

QTableWidget::item:selected {
    background-color: #b0b0b0;
    color: #000000;
}
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="searchInput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>561</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit#searchInput {
  color:#1B0633;
    border: 3px solid #8d4d95;
    border-radius: 10px;
        font-size: 17px;
font-weight: 570;
padding: 0px 8px 0px 8px;
background-color: #E8D4F4 ;
}
</string>
   </property>
   <property name="placeholderText">
    <string>🔍Search by student ID or name...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>240</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#searchButton {
  border: 1.5px solid #B89FD0;
    border-radius: 10px;
    padding: 6px;
    font-size: 16px;
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                            stop:0 rgb(95, 2, 95),
                            stop:1 rgb(216, 181, 221));
color:#e4c9f2;
font-weight:700;
}

/* Hover effect */
QPushButton#searchButton:hover {
   background-color: #69166f;
}

</string>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#label{
     background-color: #3E0066;
  color: #E6CCE7;
  border: 2px solid #E7A9A9;
  font-weight: bold;
  font-size: 20px;
  text-align: center;
}</string>
   </property>
   <property name="text">
    <string>RESULT ENTRY</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
